src/components/ClipboardCopy.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from 'react';
+import { CopyToClipboard } from 'react-copy-to-clipboard';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import styled from 'styled-components';
+
+const StyledDiv = styled.div`
```

Comment:
Let's add `cursor: pointer;` here so we have the hand mouse curson when hovering on the button.